common/sagas/wallet.js

```diff
@@ -19,41 +20,52 @@ import {
 import { BaseNode } from 'libs/nodes';
 import { getNodeLib } from 'selectors/config';
 import { getWalletInst, getTokens } from 'selectors/wallet';
-
 import { determineKeystoreType } from 'libs/keystore';
+import TransactionSucceeded from 'components/ExtendedNotifications/TransactionSucceeded';
+import type { BroadcastTxRequestedAction } from 'actions/wallet';
 
 function* updateAccountBalance() {
-  const node: BaseNode = yield select(getNodeLib);
-  const wallet: ?BaseWallet = yield select(getWalletInst);
-  if (!wallet) {
-    return;
+  try {
+    const node: BaseNode = yield select(getNodeLib);
+    const wallet: ?BaseWallet = yield select(getWalletInst);
```

Comment:
Swap order of these two, since we return if no `wallet` and it doesn't look like that has a dependency on node existing, would make sense to return without setting node if not needed.